.vscode/settings.json

```diff
@@ -0,0 +1,5 @@
+{
+	"editor.defaultFormatter": "esbenp.prettier-vscode",
+	"editor.formatOnSave": true,
+	"eslint.validate": ["javascript", "typescript", "typescriptreact"]
+}
```

Comment:
I am always unsure if this is something we should impose this way? Granted I think there is a HUGE benefit in having this occur by default. I just wonder if someone goes to change their vscode config, if it would be annoying to have to always deal with trying not to commit it